README.md

```diff
@@ -4,22 +4,16 @@
 
 * Google Chrome v 59.0.3071.115+
 
-## MetaMask plugin setup
+## MetaMask/Nifty Wallet extensions setup
 
-* Connect to POA Network in MetaMask (See [Connect to POA Network via MetaMask](https://github.com/poanetwork/wiki/wiki/POA-Network-on-MetaMask)) or Nifty Wallet extensions (See [Connect to POA Network via Nifty Wallet](https://github.com/poanetwork/wiki/wiki/POA-Network-on-Nifty-Wallet))
+* Connect to POA Network in MetaMask extension (See [POA Network on MetaMask](https://github.com/poanetwork/wiki/wiki/POA-Network-on-MetaMask)) or Nifty Wallet extension (See [POA Network on Nifty Wallet](https://github.com/poanetwork/wiki/wiki/POA-Network-on-Nifty-Wallet))
 
 * Import your initial key to MetaMask or to Nifty Wallet extensions: browse keystore file, received from invitation, and enter password for the key (See Importing of keys section [in MetaMask](https://github.com/poanetwork/wiki/wiki/POA-Network-on-MetaMask#importing-of-keys) or [in Nifty Wallet](https://github.com/poanetwork/wiki/wiki/POA-Network-on-Nifty-Wallet#importing-of-keys)).
```

Comment:
The link for `Nifty Wallet` here should point to https://github.com/poanetwork/wiki/wiki/POA-Network-on-Nifty-Wallet#importing-keys

Reply:
Thanks, fixed the link